### PR TITLE
Add auth on upload.ts | Implement Docker Healthcheck

### PR DESCRIPTION
**Pull request by yJulian**

### CODE REVIEW

**Review by Voakie — APPROVED** [review]
👆😏👍

File: src/routes/upload.ts [hunk 1]
```diff
@@ -3,14 +3,22 @@ import { os } from "../structs/database";
...
 import * as crypto from "crypto";
+import { User } from "../models/user.schema";
 
 const router = Router();
 const bucketName = "";
 
-router.post("/upload", (req, res) => {
-  const userToken = req.body.token;
+router.post("/upload", async (req, res) => {
+  const token = req.body.token;
```
Comment (DeveloperTK) [inline src/routes/upload.ts:12]: Please check if a token was provided: `req.body.hasOwnProperty("token")`

**Review by DeveloperTK — CHANGES_REQUESTED** [review]
^^

File: src/routes/upload.ts (hunk 1 above)
Comment (DeveloperTK) [reply src/routes/upload.ts:12]: And please leave the route empty, the `/upload` endpoint is already provided by the router

**Review by DeveloperTK — APPROVED** [review]

**Pull request merged**